include/library/spdm_lib_config.h

```diff
@@ -12,6 +12,18 @@
 #define LIBSPDM_FIPS_MODE 0
 #endif
 
+/* DSP0277 defines 64bit sequence number. The max number is 0xFFFFFFFFFFFFFFFFull.
+ * https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aead-limits describes
+ * how to limit the use of keys in order to bound the advantage given to an attacker.
+ *
+ * The integritor can override the default value, such as 0xFFFFFFFF (32bit) or 0xFFFFFF (24bit).
+ * If KEY_UPDATE is not sent before the LIBSPDM_MAX_SPDM_SESSION_SEQUENCE_NUMBER is reached,
+ * the SPDM session will be terminated.
+ */
+#ifndef LIBSPDM_MAX_SPDM_SESSION_SEQUENCE_NUMBER
```

Comment:
This should be settable at runtime. The default value should be `UINT64_MAX` but Integrator can override it at runtime.

Reply:
agree. updated.